### PR TITLE
Add rules.mk to ortho_4x12/buswerks keymap

### PR DESCRIPTION
**Pull request by lbussell**
Added rules.mk file to my personal keymap and tweaked the layout a bit.

### CODE REVIEW

File: layouts/community/ortho_4x12/buswerks/rules.mk [hunk 1]
```diff
@@ -0,0 +1,9 @@
+BOOTMAGIC_ENABLE  = no       # Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE   = yes       # Mouse keys(+4700)
+EXTRAKEY_ENABLE   = yes       # Audio control and System control(+450)
+CONSOLE_ENABLE    = no         # Console for debug(+400)
+COMMAND_ENABLE    = no        # Commands for debug and configuration
+TAP_DANCE_ENABLE  = no
+AUDIO_ENABLE      = yes
+
+MACROS_ENABLED    = no
```
Comment (drashna) [inline layouts/community/ortho_4x12/buswerks/rules.mk]: :D

File: layouts/community/ortho_4x12/buswerks/keymap.c [hunk 2]
```diff
@@ -7,6 +7,7 @@ extern keymap_config_t keymap_config;
 #define _DVORAK 2
```
Comment (drashna) [inline layouts/community/ortho_4x12/buswerks/keymap.c]: You may want to consider using an enum for the layers here, so you don't have to manage then numbers here.

**Comment by drashna** [comment]
Thanks!

**Comment by drashna** [comment]
Also, I noticed that you're using the master branch.   You shouldn't use the master, ever.  It makes things harder to keep up to date.

In fact, you should check out this guide: https://docs.qmk.fm/#/newbs_best_practices

**Comment by lbussell** [comment]
I see, thank you. I'll make sure to follow that guide next time.

**Pull request merged**